# Generated for NaoQi method ALFrameManager.onPythonPrint
# Print Python message in Choregraphe or logger. Should not be called manually.

# Message to print
string message
---


